File: 561.array-partition.java
/*
 * @lc app=leetcode id=561 lang=java
 *
 * [561] Array Partition
 */

// @lc code=start

import java.util.Arrays;

class Solution {
    public int arrayPairSum(int[] nums) {
        Arrays.sort(nums);

        int res = 0;

        for (int i = 0; i < nums.length; i+=2) {
            res += nums[i];
        }

        return res;
    }
}
// @lc code=end

File: 561.array-partition.py
#
# @lc app=leetcode id=561 lang=python3
#
# [561] Array Partition
#

# @lc code=start
class Solution:
    def arrayPairSum(self, nums: List[int]) -> int:
        nums.sort()
        res = 0
        for i in range(0, len(nums) - 1, 2):
            res += nums[i]
        
        return res
# @lc code=end

